packages/devtools_app/lib/src/screens/profiler/cpu_profile_columns.dart

```diff
@@ -105,15 +105,15 @@ class SourceColumn extends ColumnData<CpuStackFrame> {
   SourceColumn() : super.wide('Source', alignment: ColumnAlignment.right);
 
   @override
-  dynamic getValue(CpuStackFrame dataObject) => dataObject.processedUrl;
+  dynamic getValue(CpuStackFrame dataObject) => dataObject.packageUri;
 
   @override
   String getDisplayValue(CpuStackFrame dataObject) {
-    return dataObject.processedUrl;
+    return dataObject.packageUri;
```

Comment:
we may need to add the sourceLine number here. I think processedUrl contained this before

Reply:
@kenzieschmoll 
Would maintaining the sourceLine here: https://github.com/flutter/devtools/pull/3932/files/9c7af2f5ca37c22be0486f582c08f2a06741f0a6#diff-8ea419af3b96202def061b93440e5b119c49aba8132769f83c9289086559704dR586
be ok?

Reply:
instead of adding it there, why don't we just make `packageUriWithLineNumber` or similar a getter on CpuStackFrame and then the getter can add the source line value if present. It may be a little confusing if another place in the code tries to grab the 'packageUri' and wouldn't be expecting a line number to be on the end of it.

Reply:
gosh I was working on this earlier and thinking the exact same thing! :O

I will make it happen :D

Reply:
I've added it as a getter for now to avoid some redundancies